File: src/ui/app/root/application/models/Role/SecurityClaimupd.ts
export interface ISecurityClaimUpd {
    securityClaimId: string;
    description: string;
    enabled: boolean;
    origin: string;
    validationPattern: string;
    createdBy: number;
    createdOn: Date;
    lastUpdatedBy?: number;
    lastUpdatedOn?: Date;
}

File: src/data/Model/SecurityClaimUpd.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;

namespace Cms360.Data.Model
{

    
    public class SecurityClaimUpd
    {        
        [Key]
        public string SecurityClaimId { get; set; }
        public string Description { get; set; }
        public bool Enabled { get; set; }
        public string Origin { get; set; }
        public string ValidationPattern { get; set; }
        public long CreatedBy { get; set; }
        public DateTime CreatedOn { get; set; }
        public long? LastUpdatedBy { get; set; }
        public DateTime? LastUpdatedOn { get; set; }
    }
}